eu.stethewwolf.gresistor.json

```
@@ -0,0 +1,32 @@
+{
+    "app-id": "eu.stethewwolf.gresistor",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "42",
+    "sdk": "org.gnome.Sdk",
+    "command": "gresistor3",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland"
+    ],
+    "modules": [
+        {
+            "name": "gresistor",
+            "buildsystem": "simple",
+            "build-commands": [
+                "python3 setup.py install --prefix=${FLATPAK_DEST}",
+                "install -Dm644 eu.stethewwolf.gresistor.desktop -t ${FLATPAK_DEST}/share/applications",
+                "install -Dm644 gresistor3/pixmaps/48x48/eu.stethewwolf.gresistor.png  -t ${FLATPAK_DEST}/share/icons/hicolor/48x48/apps/",
+                "install -Dm644 gresistor3/pixmaps/128x128/eu.stethewwolf.gresistor.png  -t ${FLATPAK_DEST}/share/icons/hicolor/128x128/apps/",
+                "install -Dm644 eu.stethewwolf.gresistor.metainfo.xml -t ${FLATPAK_DEST}/share/app-info/xmls/eu.stethewwolf.gresistor.xml"
```

This is not where it goes.

```suggestion
                "install -Dm644 eu.stethewwolf.gresistor.metainfo.xml -t ${FLATPAK_DEST}/share/metainfo/"
```

also -t always indicate a directory.